projects/plugins/jetpack/_inc/client/lib/plans/constants.js

```diff
@@ -49,6 +49,10 @@ export const PLAN_HOST_BUNDLE = 'host-bundle';
 export const PLAN_WPCOM_ENTERPRISE = 'wpcom-enterprise';
 export const PLAN_VIP = 'vip';
 export const PLAN_CHARGEBACK = 'chargeback';
+export const PLAN_JETPACK_SOCIAL_BASIC = 'jetpack_social_basic_yearly';
+export const PLAN_JETPACK_SOCIAL_BASIC_MONTHLY = 'jetpack_social_basic_monthly';
+export const PLAN_JETPACK_SOCIAL_PREMIUM = 'jetpack_social_premium_yearly';
```

Comment:
Are we adding Premium here to prevent having to do this again in a few weeks? That's probably ok, as long as we can decide on the text, icon, slug name etc. Otherwise we'll need to revisit it anyway.

Reply:
Yup, that's the reason. Didn't want to come back and wait for a release for after we launch premium, but we probably have another release between and now then, so perhaps we can exclude premium for now

Reply:
You're right, I wanted to not wait for another release when we launch premium plans, but makes sense not to add something that's not actually ready yet and won't be for a bit.